git/clone_test.go

```diff
@@ -0,0 +1,93 @@
+package git
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestCloneRepo(t *testing.T) {
```

Comment:
I'm not crazy about test cases needing network access to run.  Is there any way we could make the fetching use a fake network connection? Maybe a better way to ensure clone doesn't break in the real world is to just use dgit to clone something arbitrary under travis

Reply:
Agreed, but I was thinking that this would cover the primary purpose of dgit, which to allow it to "go get" something where GitHub is by far the most popular place for people to "go get" stuff. I'm not sure how far we would want to go to mock out an https or ssh connection. Do you have something in mind for this?

Reply:
No, that's why I was thinking maybe just cloning something in Travis instead of a Go test might be a good idea.

Reply:
:+1 Ah, I understand now. Okay, I'll try hooking that up.